config.example.yml

```diff
@@ -85,4 +85,4 @@ logging:
     level: INFO
     format: "%(asctime)s|||%(levelname)-8s|%(name)s|%(message)s"
     datefmt: "%Y-%m-%d %H:%M:%S"
-  ai_instrumentation_key: "APPLICATION_INSIGHTS_INSTRUMENTATION_KEY"
\ No newline at end of file
+  ai_instrumentation_key: "APPLICATION_INSIGHTS_CONNECTION_STRING"
```

Comment:
Missing end of the file newline.

Reply:
`"smart_sampling"` should be added to the list: 

```
own_packages: ["__main__", "blurring_as_a_service", "yolov5", "evaluate_with_cvt_metrics", "validate_model",
                 "evaluate_with_coco", "detect_and_blur_sensitive_data", "move_files", "split_workload"]
```
                 
did it work without it when you tested it?